internal/provider/resource_vault_cluster_test.go

```diff
@@ -0,0 +1,136 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-hcp/internal/clients"
+)
+
+var (
+	testAccVaultClusterConfig = fmt.Sprintf(`
+resource "hcp_hvn" "test" {
+	hvn_id         = "test-hvn"
+	cloud_provider = "aws"
+	region         = "us-west-2"
+}
+	
+resource "hcp_vault_cluster" "test" {
+	cluster_id            = "test-vault-cluster"
+	hvn_id                = hcp_hvn.test.hvn_id
+	min_vault_version     = "v1.7.0"
```

Comment:
Specifying this might make the test kind of brittle as eventually 1.7.0 won't be a supported initial version. Maybe it'd be better to leave it out?